db/query.go

```diff
@@ -512,11 +512,12 @@ func (context *DatabaseContext) QueryTombstones(olderThan time.Time, limit int,
 	// View Query
 	if context.Options.UseViews {
 		opts := Body{"stale": "ok"}
-		opts[QueryParamStartKey] = 1
-		opts[QueryParamEndKey] = olderThan.Unix()
 		if limit != 0 {
+			opts = Body{"stale": false}
```

Comment:
@adamcfraser Maybe I should evaluate the `consistencyMode` parameter instead of relying on limit?

Reply:
Yes I think you're right - we should just be using that, and it looks like that's already doing the right thing (always requestPlus).

Reply:
Updated to change the param to a stale bool that I can use for both Views and N1QL Query.

It looks like there were unused params in the N1QL query that were probably copy/paste errors. Give it a look and see if you disagree.